jobs/Frontend/movieDatabase/src/views/main.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {FC, useEffect, useState} from 'react';
+import 'purecss/build/pure.css'
+
+import {getMovieById, getMoviesByTitle} from "../utils/moviesService";
+import {MoviesList} from "../components/moviesList";
+import './main.less'
+import {MovieDetails, MovieListItem} from "../types/movies";
+import {MovieDetail} from "../components/movieDetail";
+
+export const MainView: FC = () => {
+    const [movies, setMovies] = useState<MovieListItem[]>()

```

Comment:
This is actually fine, it's one of its overloads where if you call `useState<MyType>()` without an initial value, it will the state generated will be of type `MyType | undefined`.

![Screenshot_21](https://user-images.githubusercontent.com/21278533/146348501-0fce3f5c-3a9e-46fe-b3dc-fe948289d425.png)

Depends how you want to handle empty `movies`.